webpack/assets/javascripts/foreman_toast_notifications.js

```diff
@@ -0,0 +1,41 @@
+import * as ToastActions from './react_app/redux/actions/toasts';
+import store from '../javascripts/react_app/redux';
+import $ from 'jquery';
+/* eslint-disable max-params  */
+
+export function notify(message, type, link, sticky) {
+  showToast({
+    message,
+		type,
```

Comment:
the only field i can think of validating is the "type"... what is the logic for that?